backend-api/src/functions/testUtils/mockJwtBuilder.ts

```diff
@@ -3,11 +3,11 @@ import jose from "node-jose";
 
 export class MockJWTBuilder {
```

Comment:
Once this PR is merged I need to go back to this `MockJWTBuilder` and change it a bit. I didn't want to break the credential tests even more so I tried not to change it much.